e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -238,6 +238,54 @@ class RegistrationDetails {
     ).toContain(userName);
     expect(await timeStampLocator.textContent()).toContain(date);
   }
+
+  async addNote(actions: string, addNote: string) {
+    await this.page.getByText(actions).click();
+    await this.page.getByText(addNote).click();
+  }
+
+  async writeNote(placeholder: string, note: string, buttonName: string) {
+    const okButton = this.page.getByRole('button', {
+      name: buttonName,
+    });
+    await this.page.getByPlaceholder(placeholder).fill(note);
+    await this.page.waitForLoadState('networkidle');
+    await expect(okButton).toBeEnabled();
+
+    for (let i = 0; i < 2; i++) {
```

Comment:
I like the idea of the for loop to avoid repetition, but I think maybe there could be some value in manually repeating, and adding a comment with each repetition explaining what is happening. Because I look at this and I wonder "why are we doing the same thing twice?

Reply:
Yeah.. User adds note and after accepting note in the first screen then he is prompted with "saved successfully" pop-up that he has to click "ok" on it. so it is just double "Ok"button for adding one note. Would you say double command clicking Ok and waiting would be clearer in this case?